rsvp_commands.py

```diff
@@ -564,7 +569,9 @@ def run(self, events, *args, **kwargs):
     if event['description']:
         summary_table += '**What**|%s\n' % event['description']
 
+    date = datetime.datetime.strptime(event['date'], '%Y-%m-%d').date()
     summary_table += '**When**|%s @ %s\n' % (event['date'], event['time'] or '(All day)')
+    summary_table += '**Day**|%s \n' % (ISOWEEKDAY.get(date.isoweekday()))
```

Comment:
Another way to accomplish this would be to replace `(ISOWEEKDAY.get(date.isoweekday()))` with `date.strftime("%A")`